### PR TITLE
Add permissions option

### PR DESCRIPTION
**Pull request by maeldebon**
Related issue: #31 
Add a --permissions-path option

### CODE REVIEW

File: substra/cli/interface.py [hunk 1]
```diff
@@ -148,6 +148,17 @@ def click_option_verbose(f):
     )(f)
 
 
+def click_option_permissions(f):
+    """Add permissions file path to command."""
+    return click.option(
+        '--permissions-path',
+        'permissions',
+        type=click.Path(exists=True, dir_okay=False), callback=load_json_from_path,
```
Comment (jmorel) [inline substra/cli/interface.py]: Could you move callback to its own line just to make it clear it's not part of the type?

File: substra/cli/interface.py [hunk 2]
```diff
@@ -148,6 +148,17 @@ def click_option_verbose(f):
...
+        metavar="PATH",
```
Comment (jmorel) [inline substra/cli/interface.py]: I'm not sure but I think metavar only work with arguments, not options, could you double check against the documentation?

Comment (maeldebon) [reply substra/cli/interface.py]: Indeed, it was a mistake

**Comment by maeldebon** [comment]
As written in #31, this PR is not required

**Pull request closed without merging**